README.rst

```diff
@@ -106,7 +106,9 @@ file format:
         log_dir: path/to/log_dir
         bin_dir: path/to/bin_dir
         inc_dir: path/to/inc_dir
-        log_maxsize: num (MB)
```

Comment:
Why did you decide to remove `log_maxsize`?

Reply:
Accidental remove.